src/ontoform/transformers/gnomad.py

```diff
@@ -0,0 +1,16 @@
+import gzip
+from typing import BinaryIO
+
+from loguru import logger
+
+from ontoform.util import SupportedFormats
+
+
+def transform(src: BinaryIO, dst: BinaryIO, format: SupportedFormats) -> None:
+    """Take for e.g. a bgzipped file and write it to a gzip file."""
+    logger.info(
+        f"transforming {src.name} to {dst.name}, ignoring format: {format.name}"
```

Comment:
No need to 
```suggestion
        f'transforming to gzip, ignoring format argument {format.name}'
```

No need to show source and destination here, they are already logged in [the caller](https://github.com/opentargets/ontoform/blob/main/src/ontoform/main.py#L57-L58).